docs/redesign-specs/DB_Schema/database.md

```diff
@@ -0,0 +1,199 @@
+**ASSUMPTION:** A reliable underlying technology that provides 100% availablity of the database, following are some examples of the relations between different tables. 
+
+
+All the tables in the BMI database are listed bellow: 
+
+## User 
+
+1. An entity using BMI. 
+2. Users will be registered to BMI by an admin via the API after BMI has been deployed. 
+
+* id : Id of the user (primary key)
+* name : Name of the user (unique key) 
+* type : Type of user (ordinary user or admin)
+* quota : Quantity of datastore disk-space allocated to a user (GB)

```

Comment:
Is this for the disk space?  I am confused by figure, where the datastore/storage provider is coupled with the database.  I think that storage provider should be closely associated with the iscsi service.  You really want a single driver for communicating with storage to clone volumes... and to set up ISCSI end points for those volumes.  For quota, is this the absolute size of the volumes, or is this the new storage incurred due to the clone; in many case the latter will be orders of magnitude less